gemrb/GUIScripts/GUIClasses.py

```diff
@@ -212,8 +212,11 @@ def DeleteControl(self, view):
 			view = self.GetControl (view)
 		RemoveView (view, True)
 
-	def GetControl(self, newID):
-		return GetView(self, newID)
+	def GetControl(self, cid):
+		if cid is not None and cid >= 0: # FIXME: some GUIScript functions are still returning values of -1 instead on None
```

Comment:
I suggest we print the stack trace here, so the negative providers will be easier to track down.